src/scanner/custom-rules/display-accessible-names.ts

```diff
@@ -14,55 +15,30 @@ export const accessibleNamesConfiguration: RuleConfiguration = {
     ],
     rule: {
         id: accessibleNamesCheckId,
-        selector: createSelector(),
+        selector:
```

Comment:
It would be good to add a comment briefly explaining/linking to how this list was created